data/styles/vehicle/include/Icons.mapcss

```diff
@@ -482,7 +484,9 @@ area|z15[historic=memorial]
 node|z16-[historic=monument],
 area|z16-[historic=monument],
 node|z16-[historic=memorial],
-area|z16-[historic=memorial]
+area|z16-[historic=memorial],
+node|z17-[cemetery=grave],
+area|z17-[cemetery=grave],
```

Comment:
I suggest not to add it to the vehicle style as its overloaded with details already.
Let's start to treat it as a separate style we're developing for motorists with a focus on roads / vehicle amenities and reduced distraction from generic POIs.

Reply:
Ok